unit1/lesson12.rst

```diff
@@ -0,0 +1,41 @@
+Unit 1 - Lesson 12 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Create static HTML pages using a variety of different tags inside the body.
+- Create an account on Github and create hello world repository.
+- Understand what Git is and why programmers use it.
+
+Objectives
+----------
+
+HTML
+^^^^
+
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
+- Define HTML attributes and how they are used
+- Describe how whitespace in HTML documents effects rendering
+- Describe inline and block display; for common elements identify their default display
+- Describe HTML document structure and associated elements: doctype, <html>, <head>, <body>, <title>
+- Write a properly-structured document head.
+- Describe the purpose of, and use additional HTML tags: <ul>, <ol>, <li>, <table>
+- Write HTML comments
+
+Git
+^^^
+
+- Describe the main functionality provided by a version control system (VCS)
+- Describe the practical applications of this functionality: revision history, collaboration, backup
+- Explain what a repository is
+- Explain what a commit is
+- Describe the basic commit flow and commands: status, add, commit, log
+- Initialize a new Git repository
+- Commit to a local repository
+- Clone a project from a remote Git repository
+- Explain the two Git file status: tracked and untracked
+- Explain what types of changes should be within a commit
```

Comment:
Will git push/pull be addressed here or in the next lesson?

Reply:
Git push will be part of lesson 12, but pull won't be used until lesson 13.